server/helpers/valueAddedServices.js

```diff
@@ -0,0 +1,26 @@
+import { VALUE_ADDED_SERVICE_ACCESS_LEVEL } from './constants';
+
+const VALUE_ADDED_SERVICES = {
```

Comment:
If the VAS is empty, lets save the values here in the database so that the admin can add more without requiring human intervention.

```suggestion
const INITIAL_VAS = {
```